src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h

```diff
@@ -234,27 +235,11 @@ typedef GUID xuid_t;
 typedef unsigned char xuid_t[16];
 #endif
 
-typedef struct _XU_MEM_TOPO_DATA {
-
-    UCHAR m_type; //enum corresponding to mem_type.
-    UCHAR m_used; //if 0 this bank is not present
-    union {
-        ULONGLONG m_size; //if mem_type DDR, then size in KB;
-        ULONGLONG route_id; //if streaming then "route_id"
-    };
-    union {
-        ULONGLONG m_base_address;//if DDR then the base address;
-        ULONGLONG flow_id; //if streaming then "flow id"
-    };
-    UCHAR m_tag[16]; //DDR: BANK0,1,2,3, has to be null terminated; if streaming then stream0, 1 etc
-
-} XU_MEM_TOPO_DATA, *PXU_MEM_TOPO_DATA;
-
 typedef struct _XOCL_MEM_TOPOLOGY_INFORMATION {
 
     ULONG        MemTopoCount;
 
-    XU_MEM_TOPO_DATA MemTopo[XOCL_MAX_DDR_BANKS];
+    struct mem_data MemTopo[XOCL_MAX_DDR_BANKS];
```

Comment:
Isn't this wrong?     There is no limit to the number of banks.  
Also, should this be per xclbin.h, e.g. struct mem_topology should be used from xclbin.

Reply:
Yes, I agree this should be per xclbin data struct. I talked with Igor and he mentioned, this was an oversight and he will fix it.

Reply:
Fixed. I removed XOCL_MEM_TOPOLOGY_INFORMATION from XoclUser_INTF.h and made references to struct mem_topology of xclbin.h